src/main/java/com/otto/lab2/service/impl/ParsingServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.otto.lab2.service.impl;
+
+import com.otto.lab2.model.Sentence;
+import com.otto.lab2.service.ParsingService;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParsingServiceImpl implements ParsingService {
+
+    @Override
+    public List<Sentence> parseText(String input) {
+
+        String splitRegEx = "((?<=\\.)|(?<=!)|(?<=\\?))";
+
+        List<String> sentencesFromInput = Arrays.stream(input.split(splitRegEx)).
```

Comment:
format each method of stream starts from new line. Also better extract it to separate method